lib/pquot.gd

```diff
@@ -61,10 +61,11 @@ DeclareGlobalFunction( "AbelianPQuotient" );
 ##  most <M>p^{256}</M>. If the parameter <A>logord</A> is present, it will
 ##  compute with factor groups of order at most <M>p^{<A>logord</A>}</M>.
 ##  If this parameter is specified, then the parameter <A>c</A> must also be
-##  given.  The present
-##  implementation produces an error message if the order of a
-##  <M>p</M>-quotient exceeds <M>p^{256}</M> or <M>p^{<A>logord</A>}</M>,
-##  respectively.
+##  given. If the order of a <M>p</M>-quotient exceeds <M>p^{256}</M>
```

Comment:
Is `256` an additional limit, or just the default value for `logord` ?

Reply:
it is the default value, not an additional limit (at least what I can tell from the code and how it behaves on input groups that exceed 256 generators). Probably we should rephrase it. For now, I tried to stay as close to the original documentation as possible